scripts/zones/Chocobo_Circuit/npcs/Amaduralle.lua

```diff
@@ -1,7 +1,8 @@
 -----------------------------------
 -- Area: Chocobo_Circuit
---  NPC: Amaduralle
+-- NPC: Amaduralle
 -- Standard Info NPC
+-- !pos -369.286 -4.000 -464.873
```

Comment:
Can you make sure each of these commands also has the zone number, so you can copy/paste it and go to the NPC from anywhere